src/tests/dbus-tests/test_loop.py

```diff
@@ -0,0 +1,99 @@
+#!/bin/python3
+
+import unittest
+import storagedtestcase
+import subprocess
+import time
+import dbus
+import glob
+import os
+
+class StoragedLoopDeviceTest(storagedtestcase.StoragedTestCase):
+    """This is SwapSpace related functions unit test"""
+
+    LOOP_DEVICE_FILENAME = 'loop_device.img'
+
+    def get_device(self, dev_name):
+        dev = self.get_object('', '/block_devices/' + os.path.basename(dev_name))
+        return dev
+    #enddef
+
+
+    def get_loop_iface(self, device):
+        iface = dbus.Interface(device, dbus_interface=self.iface_prefix + '.Loop')
+        return iface
+    #enddef
+
+
+    def setUp(self):
+        # create a file and fill it with some data
+
+        self.FNULL = open(os.devnull, 'w')
+
+        subprocess.call(['dd', 'if=/dev/zero', 'of=%s'%self.LOOP_DEVICE_FILENAME,
```

Comment:
Please use `run_command` method from `StoragedTestCase` instead of `subprocess.*`. Feel free to adjust the method (or add a new one) if you need different functionality.